### PR TITLE
Enhance the MPASSIT task so it works with both CONUS3km and CONUS12km domains

### PR DESCRIPTION
**Pull request by hu5970**
1) Update MPASSIT fix files so all domains use the same list files.
2) Update namelist to use history file for grid so it works with both CONUS3km and CONUS12km domain and different levels. 
3) Save the 1-h MPASOUT file for cycling.

Those changes are tested with CONUS3km and CONUS12km real-time MPAS runs on JET.

### CODE REVIEW

**Review by chunhuazhou — APPROVED** [review]

**Review by BenjaminBlake-NOAA — COMMENTED** [review]
@hu5970 The changes look good to me, I just have one question.  Why are the history and diag files no longer copied into COMOUT in exrrfs_save_fcst.sh?

**Comment by hu5970** [comment]
@BenjaminBlake-NOAA The history and diag files are copied to umbrella directory for post process. This change is to copy 1-h MPASOUT instead of history and diag to COMOUT directory for cycling. The original copy of history and diag is a space holder. We need to use 1-h MPASOUT for next cycle.

**Review by BenjaminBlake-NOAA — APPROVED** [review]
@hu5970 Thanks, looks good to me.

File: parm/namelist.mpassit
Comment (MatthewPyle-NOAA) [inline parm/namelist.mpassit:1]: If grid_file_input_grid nd hist_file_input_grid are defined the same, do they need to be two distinct variables?

**Review by MatthewPyle-NOAA — COMMENTED** [review]
One small question, but can definitely merge it tomorrow.

File: parm/namelist.mpassit
Comment (guoqing-noaa) [reply parm/namelist.mpassit:1]: @MatthewPyle-NOAA That's required by the MPASSIT program.

**Review by guoqing-noaa — APPROVED** [review]

File: parm/namelist.mpassit
Comment (hu5970) [reply parm/namelist.mpassit:1]: "grid_file_input_grid" is the file provides grid information. All those grid information are included in history file, which is the file defined by "hist_file_input_grid".  We need to set both the same file for now. But will make a separate "grid_file_input_grid" in the future if we want to reduce the output file size by removing grid information from it.

**Comment by hu5970** [comment]
@MatthewPyle-NOAA Only you can change the PR title, right?

**Comment by MatthewPyle-NOAA** [comment]
> @MatthewPyle-NOAA Only you can change the PR title, right?

I don't know if only I can.  Do you see an "Edit" button off to the right side near the top where the title is in a large font?   I do, and that is how I edit titles.

**Pull request merged**